corehq/apps/users/role_utils.py

```diff
@@ -23,16 +24,39 @@ class UserRolePresets:
         BILLING_ADMIN: lambda: HqPermissions(edit_billing=True),
         MOBILE_WORKER: lambda: HqPermissions(access_mobile_endpoints=True,
                                              report_an_issue=True,
-                                             access_all_locations=True)
+                                             access_all_locations=True),
     }
 
+    PRIVILEGED_ROLES = {
```

Comment:
I'm not sure why Attendance Coordinator is managed like this, but other privileged roles (Dropzone / ERM / etc.) are not?

Reply:
Again, closing the loop, Attendance Tracking is managed this way because it is the only optional feature that has its own role. I'll add a comment in the code, in case it's not immediately clear to anyone else why this is handled this way.